src/utils/email/sendEmail.ts

```diff
@@ -0,0 +1,42 @@
+import * as env from "dotenv";
+import nodemailer from "nodemailer";
```

Comment:
`nodemailer` is missing type definitions.

I can work on fixing this issue for now.